components/root/root.jsx

```diff
@@ -240,10 +251,12 @@ export default class Root extends React.Component {
             this.onConfigLoaded();
         });
         trackLoadTime();
+        document.addEventListener('keyup', this.handleTabKey);
```

Comment:
Probably better to use 'keypress' instead of 'keyup' unless you're doing that for a specific reason